newsroom/templates/wire_item.html

```diff
@@ -3,33 +3,153 @@
 {% block title %}{{ item.headline }}{% endblock %}
 
 {% block content %}
-<article>
-    <pre>{{ item.slugline }}</pre>
+<div class="content content--item-detail">
 
-    <h1>{{ item.headline }}</h1>
-    <p>{{ _('By:') }} <b>{{ item.byline }}</b> {{ item.versioncreated|datetime_long }}</p>
+    <section class="content-header">
+        <div class="content-bar navbar justify-content-between">
 
-    <div class="row">
-        <div class="col-9">
-            <div class="lead">
-                {{ item.description_html | safe }}
-            </div>
+            <span class="content-bar__menu">
+                <i class="icon--close-thin"></i>
+            </span>
 
-            {{ item.body_html | safe }}
+            <div class="wire-column__preview__buttons">
+                <span class="wire-column__preview__icon"><i class="icon--share"></i></span>
+                <span class="wire-column__preview__icon"><i class="icon--print"></i></span>
+                <span class="wire-column__preview__icon"><i class="icon--copy"></i></span>
+                <span class="wire-column__preview__icon"><i class="icon--download"></i></span>
+            </div>
         </div>
-        <div class="col">
-            <h5>{{ _('Metadata') }}</h3>
-            <dl>
-                {% if item.subject %}
-                <dt>{{ _('Category') }}</dt>
-                <dd>{{ item.subject|join(', ', attribute='name')}}</dd>
-                {% endif %}
-                {% if item.genre %}
-                <dt>{{ _('Genre') }}</dt>
-                <dd>{{ item.genre|join(', ', attribute='name')}}</dd>
-                {% endif %}
-            </dl>
+
+    </section>
+
+
+    <article class="wire-column__preview__content--item-detal-wrap">
+        <div class="wire-column__preview__content">
+            <figure class="wire-column__preview__image">
+                <span class="wire-column__preview__image-icon">
+                    <i class="icon--resize icon--white"></i>
+                </span>
+                <img src="/static/article_preview.png" width="438" height="249">
+                <figcaption class="wire-column__preview__caption">Lorem ipsum dolor sit amet, consectetur adipiscing elit</figcaption>
+            </figure>
+
+            <div class="wire-column__preview__content--item-detail-text-wrap">
+                <div class="wire-column__preview__content--item-detail-item-text">
+                    <span class="wire-column__preview__slug">{{ item.slugline }}</span>
+                    <h2 class="wire-column__preview__headline">{{ item.headline }}</h2>
+                    <p class="wire-column__preview__author">{{ _('By:') }} <span class="bold">{{ item.byline }}</span> {{ item.versioncreated|datetime_long }}</p>
+                    <p class="wire-column__preview__lead">{{ item.description_html | safe }}</p>
+                    <div class="wire-column__preview__text">
+                        <p>{{ item.body_html | safe }}</p>
```

Comment:
there should be no `<p>` tag around body, it usually has `<p>` tags or `<divs>`

Reply:
Fixed